test/onnx/test_operators.py

```diff
@@ -1055,6 +1055,20 @@ def forward(self, x):
                         input_names=["x"],
                         dynamic_axes={"x": {0: "dim_0"}})
 
+    def test_batch_norm(self):
```

Comment:
Can we add this the onnxruntime test instead?
I really don't want to add more expect tests unless we don't have a better idea. In this case, I think a. expect file is not useful.

Reply:
There are several tests for batch norm for onnxruntime that does the same thing.

https://github.com/pytorch/pytorch/blob/master/test/onnx/test_pytorch_onnx_onnxruntime.py#L3563-L3610

Reply:
Please modify that file to test specifically for the mixed type case.